File: app/src/main/java/com/rakib/foodbankbd/adapters/OfferRecyclerViewHolder.java
package com.rakib.foodbankbd.adapters;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.rakib.foodbankbd.R;

public class OfferRecyclerViewHolder extends RecyclerView.ViewHolder {
    public ImageView offerImage;
    public TextView offerName;

    public OfferRecyclerViewHolder(@NonNull View itemView) {
        super(itemView);

        offerImage = itemView.findViewById(R.id.imageViewArea);
        offerName = itemView.findViewById(R.id.textViewArea);
    }
}

File: app/src/main/java/com/rakib/foodbankbd/adapters/OfferRecyclerViewHolder.kt
package com.rakib.foodbankbd.adapters

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView.ViewHolder
import kotlinx.android.synthetic.main.layout_area.view.*

class OfferRecyclerViewHolder(itemView: View) : ViewHolder(itemView) {
    var offerImage: ImageView = itemView.imageViewArea
    var offerName: TextView = itemView.textViewArea
}